README.rst

```diff
@@ -160,8 +160,29 @@ The geographic app is based on `django-loci <https://github.com/openwisp/django-
 and allows to define the geographic coordinates of the devices,
 as well as their indoor coordinates on floorplan images.
 
-This module also provides an API through which mobile devices can update
-their coordinates.
+This module also supports the following API endpoint.
```

Comment:
I think we should revert this line to how it was previously and add "See below for further information."